NBXplorer/Altcoins/Liquid/LiquidRepository.cs

```diff
@@ -192,7 +190,7 @@ protected override async Task AfterMatch(TrackedTransaction tx, IReadOnlyCollect
 					.Select(kv => (KeyPath: kv.KeyPath,
 								   Address: kv.Address as BitcoinBlindedAddress,
 								   BlindingKey: NBXplorerNetworkProvider.LiquidNBXplorerNetwork.GenerateBlindingKey(ts.DerivationStrategy, kv.KeyPath)))
-					.Where(o => o.Address != null)
+					.Where(o => o.Address != null && o.BlindingKey != null)
```

Comment:
the whole aftermatch should be skipped if unblinded, no need of hacking.

Reply:
fixed